src/utils/string.ts

```diff
@@ -31,6 +35,10 @@ export function formatDate(date: Date | string) {
 	return defaultDateTimeFormatter.format(date);
 }
 
+export function formatMilliseconds(ms: number) {
+	return `${(ms / 1000).toFixed(3)} seconds`;
```

Comment:
it is time for SINGULAR CASE
(right now this'll say 1 seconds in the rare case that it's exactly 1000 ms)